lightly/api/download.py

```diff
@@ -0,0 +1,192 @@
+import concurrent.futures
+import os
+import pathlib
+import shutil
+import threading
+import warnings
+from concurrent.futures import ThreadPoolExecutor
+from typing import List, Iterable, Union, Tuple, Dict
+
+import av
+import requests
+import tqdm
+from lightly.api import utils
+import PIL
+
+
+def download_image(url: str, session: requests.Session = None) -> PIL.Image.Image:
+    """Downloads an image from an url.
+
+    Args:
+        url: 
+            Url where image is stored.
```

Comment:
where THE image

Reply:
Furthermore, the term "stored" is ambigous. Later int this file, you use the the it for describing the OUTPUT filepath. 
Thus I would rather call it "URL where the image is downloaded from." 
Compare also with lines 112-115. 

The same applies to other docstrings of urls in this file.

Reply:
Or just call it read url to avoid any confusion